extensions/amp-video/0.1/video-cache.js

```diff
@@ -41,7 +44,10 @@ export function fetchCachedSources(videoEl, win) {
   return servicePromise
     .then((service) => service.createCacheUrl(videoUrl))
     .then((cacheUrl) => {
-      const requestUrl = cacheUrl.replace('/c/', '/mbv/');
+      const requestUrl = addParamsToUrl(cacheUrl.replace('/c/', '/mbv/'), {
+        'amp_video_host_url': Services.documentInfoForDoc(win.document)
```

Comment:
Please add the inline comment here, it should specifically mention that this is the canonical URL of the document hosting the video (and not the video host, as the parameter name is confusing)